### ISSUE TITLE
Add to HACS

### ISSUE THREAD

**Issue by codeclinic** [TarheelGrad1998/GalleryPanel#3]
Hi,

Is there any chance that you could add this to HACS? It'd make it easier to install and update.

Thanks!

**Comment by corvy** [comment]
Yes I would also suggest this. The plugin is great and exacly what I needed. Please add it to HACS! :D

**Comment by McGiverGim** [comment]
One more for this! I think is only add one file to your repo...
